api/shipping_categories.rst

```diff
@@ -0,0 +1,430 @@
+| description | Description of shipping category    |
++-------------+-------------------------------------+
+
+
+.. note::
+
+    Read more about `shipping categories`__ in a components models section.
+
+__ http://docs.sylius.org/en/latest/components/Shipping/models.html#shippingcategory
+
+Index of all shipping categories
+--------------------------------
+
+You can retrieve the full shipment categories list by making the following request:
+
+Definition
+..........
+
+.. code-block:: text
+
+    GET /api/shipping-categories/
+
++---------------+----------------+-------------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                       |
++===============+================+===================================================================+
+| Authorization | header         | Token received during authentication                              |
++---------------+----------------+-------------------------------------------------------------------+
+| page          | query          | *(optional)* Number of the page, by default = 1                   |
++---------------+----------------+-------------------------------------------------------------------+
+| limit         | query          | *(optional)* Number of items to display per page, by default = 10 |
++---------------+----------------+-------------------------------------------------------------------+
+
+
+Example
+.......
+
+.. code-block:: bash
+
+    curl http://sylius.dev/api/shipping-categories/
+        -H "Authorization: Bearer MWExMWM0NzE1NmUyZDgyZDJiMjEzMmFlMjQ4MzgwMmE4ZTkxYzM0YjdlN2U2YzliNDIyMTk1ZDhlNDYxYWE4Ng"
+        -H “Accept:application/json”
+
+Example Response
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    STATUS: 200 OK
+
+.. code-block:: json
+
+    {
+        "page":1,
+        "limit":10,
+        "pages":1,
+        "total":2,
+        "_links":{
+            "self":{
+                "href":"\/app_dev.php\/api\/shipping-categories\/?page=1&limit=10"
+            },
+            "first":{
+                "href":"\/app_dev.php\/api\/shipping-categories\/?page=1&limit=10"
+            },
+            "last":{
+                "href":"\/app_dev.php\/api\/shipping-categories\/?page=1&limit=10"
+            }
+        },
+        "_embedded":{
+            "items":[
+                {
+                    "id":1,
+                    "code":"SC1",
+                    "name":"Regular",
+                    "_links":{
+                        "self":{
+                            "href":"\/app_dev.php\/api\/shipping-categories\/1"
+                        }
+                    }
+                },
+                {
+                    "id":2,
+                    "code":"SC2",
+                    "name":"Heavy",
+                    "_links":{
+                        "self":{
+                            "href":"\/app_dev.php\/api\/shipping-categories\/2"
```

Comment:
Please remove app_dev.php.

Also, we should add spaces between ``:`` and ``{``, ``[`` etc.